docs/en/modules/net_info.md

```diff
@@ -0,0 +1,79 @@
+# net_info Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2017-01-21 | [smcl](http://blog.mclemon.io/esp8266-contributing-to-the-nodemcu-ecosystem) | [wolfgangr](https://github.com/wolfgangr/nodemcu-firmware) | [net_info.c](../../../app/modules/net_info.c)
+
+
+This module is a stub to collect common network diagnostic and analysis tools.
+
+##net_info.ping()
+ send ICMP ECHO_REQUEST to network hosts 
+ 
+**Synopsis:**:
+```
+net_info.ping(host [, count [, callback]]) 
```

Comment:
I understand your headaches, really 😃 And don't take my review comments personally, your work is highly appreciated!
If ok from your side, I would resolve such cosmetic issues by adding commits to your branch myself.